src/sentry/static/sentry/app/utils/performance/quickTrace/quickTraceQuery.tsx

```diff
@@ -54,7 +64,10 @@ export default function QuickTraceQuery({children, event, ...props}: QueryProps)
                   const trace = flattenRelevantPaths(event, subtrace);
                   return children({
                     ...traceFullResults,
+                    isLoading: traceFullResults.isLoading || metaResults.isLoading,
+                    error: traceFullResults.error || metaResults.error,
```

Comment:
Doesn't grouping trace + meta mean that if meta broke for whatever reason the entire page would appear to error when we could've loaded the trace still?

Reply:
Hm... So the impact here is just for quick trace (trace view is not affected here), but that is a good point. I can split it out to get around this problem

Reply:
resolved in 6540a2c